scylla-cql/src/frame/response/result.rs

```diff
@@ -1241,10 +1324,22 @@ fn deser_prepared(
         as usize;
     let id: Bytes = buf[0..id_len].to_owned().into();
     buf.advance(id_len);
+    let result_metadata_id = features
+        .scylla_metadata_id_supported
```

Comment:
:warning: Fallible indexing spotted! Shouldn't this have been fixed in a previous PR? We risk a panic due to invalid indexing IIUC.